src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -68,17 +81,40 @@
         </v-card-text>
       </template>
 
-      <template v-else>
-        <v-card-text>
-          <v-container fluid>
-            <v-row no-gutters class="text-center">
-              <v-col cols="12">To request a name for a Society</v-col>
-              <v-col cols="12">please use the Societies Online website</v-col>
-              <v-col cols="12">
-                <a href="https://www.bcregistry.ca/societies/">https://www.bcregistry.ca/societies/</a>
-              </v-col>
-            </v-row>
-          </v-container>
+      <!-- Category tables - xpro (amalgamation only) -->
+      <template v-if="isAmalgamation">
+        <v-row class="pt-4">
+          <v-col cols="12">
+            <span class="copy-small">Extraprovincial:</span>
+          </v-col>
+        </v-row>
+
+        <v-card-text class="pt-4">
+          <v-row no-gutters>
+            <v-col v-for="(category, i) in tableDataXPRO" :key="`cat-xpro-${i}`">
```

Comment:
This is the second block for amalgamations only. Note that it displays Xpro data directly. (The block above, for amalgamations, displayed BC data.)

See screenshot attached to comment above for example.